src/cli/commands/generate.command.ts

```diff
@@ -0,0 +1,46 @@
+import got from 'got';
+
+import { Command } from './command.interface.js';
+import { MockServerData } from '../../shared/types/mock-server-data.type.js';
+import { TSVOfferGenerator } from '../../shared/libs/offer-generator/index.js';
+import { TSVFileWriter } from '../../shared/libs/file-writer/index.js';
+import { getErrorMessage } from '../../shared/helpers/index.js';
+
+export class GenerateCommand implements Command {
+  private initialData: MockServerData;
+
+  private async load(url: string) {
+    try {
+      this.initialData = await got.get(url).json();
+    } catch {
+      throw new Error(`Can't load data from ${url}`);
+    }
+  }
+
+  private async write(filepath: string, offerCount: number) {
+    const tsvOfferGenerator = new TSVOfferGenerator(this.initialData);
+    const tsvFileWriter = new TSVFileWriter(filepath);
+    for (let i = 0; i < offerCount; i++) {
+      await tsvFileWriter.write(tsvOfferGenerator.generate());
+    }
```

Comment:
const queue = [];
for (let i = 0; i < offerCount; i++) {
  queue.push(tsvFileWriter.write(tsvOfferGenerator.generate()))
}

await Promise.all(queue)